persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_24/persist/foo_db_script.sql

```diff
@@ -0,0 +1,7 @@
+DROP TABLE IF EXISTS ByteTest;
+
+CREATE TABLE ByteTest (
+	id INT NOT NULL,
+	binary1 BINARY NOT NULL,
```

Comment:
why `binaryOptional` field is not in the script

Reply:
updated in [a82dfeb](https://github.com/ballerina-platform/persist-tools/pull/145/commits/a82dfeb7f4ac126ec1bdc229582b746bde2f1d6f)